src/datetimepicker.test.jsx

```diff
@@ -202,4 +202,31 @@ describe('DateTimePicker', function() {
 		});
 
 	});
+
+	describe('onChangeCallback in props', () => {
+		let changeCallbackSpy;
+
+		beforeEach(() => {
+			changeCallbackSpy = spyOn(DateTimePicker.propTypes, 'onChangeCallback');
+			dateTimeComponent = TestUtils.renderIntoDocument(
+				<DateTimePicker name='start_time'
+					value={dateStr}
+					onChangeCallback={function(){}}
+					forceCalendar
+				/>
+			);
+		});
+
+		it('is called when time input changes', () => {
+			const dateTimeInputEl = TestUtils.findRenderedDOMComponentWithTag(dateTimeComponent.timeComponent, 'input');
+			TestUtils.Simulate.change(dateTimeInputEl);
+			expect(changeCallbackSpy).toHaveBeenCalled();
```

Comment:
since you know the expected arguments for the spy, check those as well

```js
expect(changeCallbackSpy).toHaveBeenCalledWith(new Date(dateStr), 'start_time');
```

You might have to use `expect.any(Date)` for the first arg of `toHaveBeenCalledWith`, but try checking against a specific date first.